components/manager/scripts/validate.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
Yeah, that could very well be the case in the future, but while it is a good idea to validate in the blueprint, the CLI cannot rely that such validation indeed occur.